pages/about.tsx

```diff
@@ -68,98 +69,131 @@ const AboutPage = () => {
         <p className="pb-8">
-          The memory is a region that only exists during the smart contract
-          execution, and is accessed with a byte offset. While all the 32-byte
-          address space is available and initialized to 0, the size is counted
-          with the highest address that was accessed. It is generally read and
-          written with <RelativeLink to="#51" title="MLOAD" /> and{' '}
-          <RelativeLink to="#52" title="MSTORE" /> instructions, but is also
-          used by other instructions like{' '}
+          Unlike EVM code, EVM memory is only accessible during smart contract
+          execution. Memory is not persistent, and ceases at the end of the call
+          context. At the start of a call context, memory is initialized to 0.
+          Reading and Writing from memory is usually done with{' '}
+          <RelativeLink to="#51" title="MLOAD" /> and{' '}
+          <RelativeLink to="#52" title="MSTORE" /> instructions respectively,
+          but can also be accessed by other instructions like{' '}
           <RelativeLink to="#F0" title="CREATE" /> or{' '}
           <RelativeLink to="#F3" title="EXTCODECOPY" />.
         </p>
       </SectionWrapper>
 
-      <SectionWrapper header={<H3>The storage</H3>} anchorKey="storage">
+      <SectionWrapper header={<H3>The Storage</H3>} anchorKey="storage">
         <p className="pb-8">
-          The storage is the persistent memory of the smart contract. It is a
-          map of the 32-byte slot to 32-byte value, and each value written is
-          kept until it is set to 0 or the contract self-destruction. Reading
-          from an unset key also returns 0. It is read and written with the
-          instructions <RelativeLink to="#54" title="SLOAD" /> and{' '}
+          Storage is a map of 32-byte slots to 32-byte values. Storage is the
+          persistent memory of smart contracts: each value written by the
+          contract is retained past the completion of a function call, unless
+          its value is changed to 0, or the callee contract of{' '}
+          <RelativeLink to="#FF" title="SELFDESTRUCT" /> instruction is
+          executed. Reading stored bytes from an unwritten key also returns 0.
+          Each contract has its own storage, and cannot read or modify storage
+          from another contract. Storage is read and written with instructions{' '}
+          <RelativeLink to="#54" title="SLOAD" /> and{' '}
           <RelativeLink to="#55" title="SSTORE" />.
         </p>
       </SectionWrapper>
 
       <SectionWrapper header={<H3>The calldata</H3>} anchorKey="calldata">
         <p className="pb-8">
-          The calldata region is the data that is sent with a transaction. In
-          the case of contract creation, it would be the constructor code. This
-          region is immutable and can be read with the instructions{' '}
+          The calldata region is the data sent to a transaction as part of a
```

Comment:
I think it would be useful to make a precision here: when a contract executes a Xcall instruction, it also creates a transaction (albeit internal one), and so there is also a calldata region in the new context.

Reply:
Added in!